tests/PHPStan/Analyser/nsrt/preg_match_shapes.php

```diff
@@ -472,3 +470,21 @@ function (string $s): void {
 		assertType("array{string, non-empty-string}", $matches);
 	}
 };
+
+function bug11323(string $s): void {
+	if (preg_match('/([*|+?{}()]+)([^*|+[:digit:]?{}()]+)/', $s, $matches)) {
```

Comment:
I would expect a test with a capturing group of `([:digit:])` which resolves to a numeric-string.

or was this intentionally not implemented with this PR?